Workspace/src/ik/src/main.cpp

```diff
@@ -13,20 +13,24 @@ main(){
     Roboarm arm(1.7, 1.0, 0.3, 0.2, -1.5, -0.2);
     std::ofstream myfile;
     myfile.open("build/ik/ee_crds.csv");
-    myfile << "x_crd,y_crd,phi,\n";
+    myfile << "eePhi,eeX,eeY,wristX,wristY,elbowX,elbowY,shoulderX,shoulderY,\n";
 
     int count = 0;
     while(count<2000){
 
         count++;
         //calculate joint velocities
-        arm.calc_velocities(0.15,0.0, 0.000); //move about 10cm/s
+        arm.calc_velocities(0.15,0.0, 0.000, arm.angleLink1, arm.angleLink2, arm.angleLink3); //move about 10cm/s
 
         //calculate EE pose 0.01s in the future and update joint positions
         arm.calc_pose(true);
 
         //save the coordinates of the EE
-        myfile << arm.eeX << "," << arm.eeY<<","<<arm.eePhi << "\n";
+        myfile <<arm.eePhi<<","<<
```

Comment:
we don't need the `eePhi` for the visualization. Also, it'd be more useful if the delimeter between different points between coordinates is different. like: `x1,y1 x2,y2`. It doesn't matter what you use as a delimiter, as long as they're different. (in this case, commas and spaces)